src/github3/orgs.py

```diff
@@ -727,18 +738,33 @@ def create_team(self, name, repo_names=[], permission="pull"):
                 repositories accessible by this team
             - ``admin`` -- members can push, pull and administer
                 repositories accessible by this team
+        :param int parent_team_id:
+            (optional), the ID of a team to set as the parent team.
+        :param str privacy:
+            (optional), options:
+
+            - ``secret`` -- (default) only visible to organization
+                owners and members of this team
+            - ``closed`` -- visible to all members of this organization
         :returns:
             the created team
         :rtype:
             :class:`~github3.orgs.Team`
         """
+        headers = (
+            self.PREVIEW_HEADERS
+            if parent_team_id or privacy == "closed"
+            else None
+        )
         data = {
             "name": name,
             "repo_names": repo_names,
             "permission": permission,
+            "parent_team_id": parent_team_id,
```

Comment:
It seems like we can't actually include this if we're not sending `hellcat-preview` media type header. Should this be conditional based on it being not `None`?

Reply:
yeah, it ignored if no `hellcat-preview`.
Fixed it, thank you. It`s added to data if it`s not None. Please, check